script/signing_helper.py

```diff
@@ -143,6 +143,11 @@ def is_chrome_branded():
             """ Not chrome branded """
             return False
 
+        @property
```

Comment:
this change is needed or the output directory will be `stable` instead of `development`